mkdd_editor.py

```diff
@@ -2250,6 +2250,21 @@ def button_side_button_action(self, option, obj=None):
             self.object_to_be_added = [libbol.KartStartPoint.new(), -1, -1]
             self.pik_control.button_add_object.setChecked(True)
             self.level_view.set_mouse_mode(mkdd_widgets.MOUSE_MODE_ADDWP)
+        elif option == "route_object":
+            new_route = libbol.Route.new()
+            forward, up, left = obj.rotation.get_vectors()
+
+            new_point_1 = libbol.RoutePoint.new()
+            new_point_1.position = obj.position + left * 500
+            new_route.points.append(new_point_1)
+
+            new_point_2 = libbol.RoutePoint.new()
+            new_point_2.position = obj.position + left * -750
```

Comment:
Question: Why 500 and -750? Why not symmetric?

Reply:
The default value for path point id is 0, so it makes sense that an object's route should be generated closer to the specific point it is assigned to. If you look at the kuribos on mac, you'll see that those are even more not symmetric.

Reply:
I see. The asymmetry made me think it was a typo or something. Ideally, the first position should match the position of the object, but I understand how having two cubes overlap is not ideal.

Reply:
I can make it something like 250 vs 750 to make it seem intentional.

Reply:
I think that'd be reasonable.